lib/Backends/Habana/Habana.cpp

```diff
@@ -93,10 +93,10 @@ static std::string getKernelName(llvm::StringRef kernelBase, ElemKind kind) {
 }
 
 /// If \p PH is an output placeholder, \returns the SaveNode.
-static SaveNode *getOutputSave(Placeholder *PH) {
```

Comment:
LGTM.  We should probably move this method to the target independent part of the compiler.

Reply:
Good idea, we've probably reimplemented this function at least a few other places.